test/conformance/api/v1/route_test.go

```diff
@@ -137,6 +137,12 @@ func TestRouteGetAndList(t *testing.T) {
 }
 
 func TestRouteCreation(t *testing.T) {
+	if test.ServingFlags.SkipTestsNotSupported {
```

Comment:
@dprotaso I couldn't get the -skip go flag to bypass multiple tests, just a single one. Is this an unacceptable temp solution until we investigate the failures?

Reply:
using env vars instead as suggested